plugins/catalog-import/src/components/ImportComponentPage.tsx

```diff
@@ -57,14 +57,44 @@ export const ImportComponentPage = ({
     <Page themeId="home">
       <Header title="Register an existing component" />
       <Content>
-        <ContentHeader title="Start tracking your component on backstage">
+        <ContentHeader title="Start tracking your component in Backstage">
           <SupportButton>
-            Start tracking your component in Backstage. TODO: Add more
-            information about what this is.
+            Start tracking your component in Backstage by adding it to the
+            software catalog.
           </SupportButton>
         </ContentHeader>
-        <Grid container spacing={3} direction="column">
-          <Grid item>
+        <Grid container spacing={3} direction="row-reverse">
+          <Grid item xs={5}>
```

Comment:
I thought these needed to be divisible by 12? I’d suggest changing to 4+8 on big screens and maybe even 12 (only one column) on the smallest screen sizes.

Reply:
According to https://material-ui.com/components/grid/, it's broken up into 12, but doesn't necessarily need to be divisible by 12.  In the sense that, there are always 12 to a grid, and any numbers that added up for columns that = 12 will be shown on a row?  That's how I understand it. Since there was still whitespace in the input box card, that's why I made it 8 and 5.  Can for sure be tweaked.

But I'll give a few reviews of what it looks like on mobile using Edge(Chrome) device emulation and throw a tweak or two in.

Reply:
Yep you understood it right, it's just that we have never used 5. 3, 4, and 6 (OK, and 12) are used in the codebase because they evenly divide things in halves/thirds/quarters.